python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_sensor_invocation.py

```diff
@@ -662,3 +669,192 @@ def asset_sensor(context):
             [july_daily_partitions.key], instance=instance, repository_def=my_repo
         )
         list(asset_sensor(ctx))
+
+
+@pytest.mark.parametrize(
```

Comment:
I kinda just put a bunch of random test cases I thought of together, this should be more structured